packages/component/src/components/image-preview-modal/index.tsx

```diff
@@ -112,6 +114,51 @@ export const ImagePreviewModal = (
   props: ImagePreviewModalProps
 ): ReactElement | null => {
   const [blockId, setBlockId] = useAtom(previewBlockIdAtom);
+
+  const handleKeyDown = useCallback(
+    (event: KeyboardEvent) => {
+      if (event.key === 'Escape') {
```

Comment:
Maybe we need to prevent default & stop propagation after any keyboard event is handled.